metakb/main.py

```diff
@@ -0,0 +1,61 @@
+"""Main application for FastAPI."""
+from fastapi import FastAPI, Query
+from fastapi.openapi.utils import get_openapi
+from metakb.query import QueryHandler
+from metakb.schemas import SearchService
+from typing import Optional
+
+app = FastAPI(docs_url='/api/v2', openapi_url='/api/v2/openapi.json')
+query = QueryHandler(uri="bolt://localhost:7687",
+                     credentials=("neo4j", "admin"))
+
+
+def custom_openapi():
+    """Generate custom fields for OpenAPI response."""
+    if app.openapi_schema:
+        return app.openapi_schema
+    openapi_schema = get_openapi(
+        title="The VICC Meta-Knowledgebase",
+        version="2.0",
```

Comment:
I think we might still be in pre-release -- maybe 2.0.0-alpha.1?

Reply:
Changed!